56.md

```diff
@@ -75,7 +74,7 @@ Client behavior
 ---------------
 
 Clients can use reports from friends to make moderation decisions if they
-choose to. For instance, if 3+ of your friends report a profile as explicit,
```

Comment:
Explicit is not a listed report type, so I replaced it with one that is.

Reply:
This NIP was never really built to be exhaustive, see #457 and #532 for discussion of the topic. A more complete classification system could be implemented with labels, nips 36 and 56 are sort of just for flagging content for manual review, although it would be fine to continue to build them out.